test/integration/patients/patient/sidebar.js

```diff
@@ -493,10 +518,18 @@ context('patient sidebar', function() {
 
     cy
       .url()
-      .should('contain', '/patient/dashboard/1');
+      .should('contain', `/patient/dashboard/${ testPatient.id }`);
   });
 
   specify('view patient modal', function() {
+    const testPatient = getPatient({
```

Comment:
I think this might be just `const testPatient = getPatient()`.   We should attempt to remove attributes from this test and those above that are not required for the tests.  We were previously over-setting because we didn't have mergejson to cover all the defaults

Reply:
I was able to remove the `sex` attribute, but the first/last name and dob attributes were needed for the test.

Reply:
Ah missed that one, looks like that one only needs to the `source: 'manual'` attribute.